CWE/map/positions.txt

```diff
@@ -9717,6 +9717,15 @@
             y = 1614.750000
         }
 
+        naval_base = {
```

Comment:
Remove this naval base, it's completely useless as ships spawned here have no way of getting out

Reply:
oh that's just new changes not having been processed, will do